interfaces/api/service/IMapsManager.h

```diff
@@ -134,53 +124,56 @@ class XPCF_CLIENTUUID("4863f2d1-023c-4095-8842-e6083e6ec54a") XPCF_SERVERUUID("2
         const std::string & mapUUID,
         SRef<SolAR::datastructure::PointCloud> & pointCloud) const = 0;
 
-    /// @brief Register a new Map Processing service to the map manager
-    /// @param[in] processingType the type of process managed by the service
-    /// @param[in] serviceURL URL of the Map Processing service
-    /// @return FrameworkReturnCode::_SUCCESS if the Map Processing service is registered, else FrameworkReturnCode::_ERROR_
-    virtual FrameworkReturnCode registerMapProcessingService(const MapProcessingType processingType,
-                                                             const std::string & serviceURL) = 0;
-
-    /// @brief Unregister a Map Processing service from the map manager
-    /// @param[in] processingType the type of process managed by the service
-    /// @param[in] serviceURL URL of the Map Processing service
-    /// @return FrameworkReturnCode::_SUCCESS if the Map Processing service is unregistered, else FrameworkReturnCode::_ERROR_
-    virtual FrameworkReturnCode unregisterMapProcessingService(const MapProcessingType processingType,
-                                                               const std::string & serviceURL) = 0;
+    /// @brief Request the map manager for information on a specific map
+    /// @param[in] mapUUID UUID of the map
+    /// @param[out] descriptorType the descriptor type used to extract descriptor for each keyframe
+    /// @param[out] mapSupportedTypes the types of available data
+    /// *           (_PointCloud = 0x01, _Keyframe = 0x02, _CovisibilityGraph = 0x04, _KFRetriever = 0x08, _CameraParemeters = 0x10)
+    /// @param[out] dataSize global size of the data structure of the map (in bytes)
+    /// @param[out] areImageSaved true if images are saved with keyframes, false otherwise
+    /// @return
+    /// * FrameworkReturnCode::_SUCCESS if the map information is available
+    /// * FrameworkReturnCode::_UNKNOWN_MAP_UUID if mapUUID is unkown
+    /// * else FrameworkReturnCode::_ERROR_
+    virtual FrameworkReturnCode getMapInfo(const std::string & mapUUID,
```

Comment:
nit: not for now, but what about defining a MapInfo struct? This would allow to add more things in it without modifying the signature of `getMapInfo()` methods.

Reply:
I agree with this: to keep in mind